FREYA-1182: Update LICENSE: add MIT license

aishling-scilifelab opened the pull request:
- The contents are covered by the Creative Commons license but the MIT license is for any software/code components.
- The code here in this repo is minimal but safer to have it.

**Question: whilst I was in there, I saw that the version of the Creative Commons license is quite old - possibly should be updated**

aishling-scilifelab committed:
Update LICENSE: add MIT license

- The contents are covered by the Creative Commons license but the
MIT license is for any software/code components.
- The code here in this repo is minimal but safer to have it.

senthil10 commented on LICENSE:
```diff
@@ -18,6 +18,30 @@ Notices:
 
+## Software
+
+Copyright (c) 2025 SciLifeLab
```
In other repos it say `SciLifeLab Data Centre`, should we use the same ?

aishling-scilifelab replied:
I didn't here because if this a collab between us and NBIS I guess SciLifeLab might cover both? Other option is to do: NBIS and SciLifeLab Data Centre.

senthil10 replied:
Makes sense 👍🏽

YvonneKallberg commented:
Reading the license I realize that it needs an update overall. It talks about the Data Stewardship Wizard, but this site is only a guide, not the tool itself. It probably made sense previously, when it was more integrated with the tool, but not now.

aishling-scilifelab commented:
@YvonneKallberg - makes sense. I am pro getting this update merged in and then any updates to the Creative Commons part and the Ts & Cs can come in different updates. Does that work for you?

aishling-scilifelab committed:
Remove reference to licensing from README.md

- The LICENSE file is the standard place to find this info.
- Having it mentioned here means there's a second place we need to
remember to update when LICENSE's contents are updated.
